lib/NewDataList/NewDataList.js

```diff
@@ -38,6 +38,10 @@ module.exports = kind({
 
 		e = event.type;
 
+		if((e === 'onSpotlightUp' && this.direction == 'horizontal') || (e === 'onSpotlightLeft' && this.direction == 'vertical')) {
```

Comment:
updated the code as per review comments.
But not sure about combining the conditions as  acceleration is relatively less frequent  I guess (in performance perspective).
Thankyou.